checkov/secrets/plugins/custom_regex_detector.py

```diff
@@ -0,0 +1,135 @@
+from __future__ import annotations
+
+import json
+import logging
+import os
+from typing import Set, Any, Generator, Pattern, Optional, Dict, Tuple, cast, List
+
+import yaml  # type: ignore
+from detect_secrets.constants import VerifiedResult
+from detect_secrets.core.potential_secret import PotentialSecret
+from detect_secrets.plugins.base import RegexBasedDetector
+import re
+
+from checkov.common.bridgecrew.platform_integration import bc_integration
+from detect_secrets.util.code_snippet import CodeSnippet
+from detect_secrets.util.inject import call_function_with_arguments
+
+
+DETECTORS_BY_CUSTOMER_CACHE: dict[str, list[dict[str, Any]]] = {}
+
+
+def get_customer_cache() -> dict[str, list[dict[str, Any]]]:
+    return DETECTORS_BY_CUSTOMER_CACHE
+
+
+def get_detectors_from_cache(customer_name: str | None) -> list[dict[str, Any]]:
+    if customer_name:
+        cache = get_customer_cache()
+        return cache.get(customer_name, [])
+    return []
+
+
+def get_detectors_from_local_file() -> list[dict[str, Any]]:
```

Comment:
Is this method called from anywhere?

Reply:
not anymore, you're right, I will delete it